### DISCUSSION TITLE
Problem with S3 missing file

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#6401] [Help] [answered]
Hi All,

I tried to save the image into s3. The file is saved in livewire-temp folder, then disappear when I click the save button.

```
                Forms\Components\FileUpload::make('attachment')
                ->label('attachment')
                ->disk('s3')
                ->preserveFilenames()
```
and I'm using  "league/flysystem-aws-s3-v3": "3.0" 

**Screen shoot when uploading** 
![image](https://user-images.githubusercontent.com/6214964/236124000-edea5c5c-7852-49df-90f2-f2c34cdd7955.png)

**At S3 Bucket, that look like that file already uploaded into livewire-temp folder**
![image](https://user-images.githubusercontent.com/6214964/236124283-06f659eb-91a9-41dd-bea3-6e5cb3888038.png)

**Screen shoot after is uploaded**
![image](https://user-images.githubusercontent.com/6214964/236124735-4c1d71fe-945b-44d1-9f5d-c382f3fd73d9.png)

**A S3 Bucket, that look the folder and file is missing ??**
![image](https://user-images.githubusercontent.com/6214964/236124626-17b220d5-c932-4b44-9ce6-f2cf771403af.png)

*At database, It look the field is updated with correctly
![image](https://user-images.githubusercontent.com/6214964/236126812-7e124f4e-a2b6-4692-8bd0-0a025213bee7.png)

I need help to solve this problem.
Thank you.

**Comment by danharrin** [comment]
If you refresh the edit page after saving, does the file upload contain the correct image?

**Reply by jigujigu** [reply]
there is no image  after I refresh the edit page.

the file also is missing on s3.

there also no error message in my browser console

**Reply by danharrin** [reply]
I don't know then, its hard to debug without an error message, especially when it works for everyone else.

**Reply by jigujigu** [reply, marked as answer]
the problem is solved.

I delete and recreate the bucket. 

thanks.
